test/llilc_runtest.py

```diff
@@ -171,7 +173,8 @@ def main(argv):
     # Run the test
     return_code = const.RunTestOK
     exclusion = os.path.join(os.path.dirname(__file__), 'exclusion.targets')
-    runtest_command = 'runtest ' + args.arch + ' ' + args.build 
+    runtest_full_path = os.path.join(runtest_dir, 'runtest')
```

Comment:
I think this is meant to also be run on Linux, which will run the runtest.sh script (whenever we get testing on Linux). I test for the cmd above because it should always be there (and is currently the only script there). I can change this, but I think the intent was to be ready for cross plat once we got testing there.